repose-aggregator/build.gradle

```diff
@@ -22,12 +22,13 @@ task copyXsds(type: Copy) {
 
 asciidoctor {
     requires 'asciidoctor-diagram'
+    options template_dirs: ["${project.projectDir}/src/docs/asciidoc/_backends/haml"]
     attributes 'source-highlighter': 'coderay',
                'stylesdir': "${project.projectDir}/src/docs/asciidoc/_stylesheets",
                'stylesheet': 'repose.css',
                'toc': 'left',
                'icons': 'font',
-               'docinfo1': ''
+               'docinfo': 'shared'
```

Comment:
I also changed this. `docinfo1` is just an alias for what I replaced it with, but since the attributes were added later as a replacement (since they are more descriptive) I switched to using those.